lib/gpu/src/lib.rs

```diff
@@ -34,6 +37,18 @@ pub use shader::*;
 #[cfg(test)]
 mod basic_test;
 
+#[cfg(any(test, feature = "testing"))]
+pub static GPU_PANIC_MESSENGER: LazyLock<Box<dyn DebugMessenger>> =
+    LazyLock::new(|| Box::new(PanicIfErrorMessenger {}));
+
+#[cfg(any(test, feature = "testing"))]
+pub static GPU_TEST_INSTANCE: LazyLock<std::sync::Arc<Instance>> = LazyLock::new(|| {
+    Instance::builder()
+        .with_debug_messenger(GPU_PANIC_MESSENGER.as_ref())
+        .build()
+        .unwrap()
+});
```

Comment:
Nit: why not just inline it?

```suggestion
pub static GPU_TEST_INSTANCE: LazyLock<std::sync::Arc<Instance>> = LazyLock::new(|| {
    Instance::builder()
        .with_debug_messenger(Box::new(PanicIfErrorMessenger {}))
        .build()
        .unwrap()
});
```

Reply:
Cause 'with_debug_messenger' takes reference and don't take ownership. Fixed, now 'with_debug_messenger' works with Box instead of reference.